src/named.rs

```diff
@@ -86,15 +86,15 @@ pub enum NamedChain {
     #[cfg_attr(feature = "serde", serde(alias = "koi"))]
     Koi = 701,
 
-    #[strum(to_string = "bsc", serialize = "binance-smart-chain")]
-    #[cfg_attr(feature = "serde", serde(alias = "bsc", alias = "binance-smart-chain"))]
-    BinanceSmartChain = 56,
-    #[strum(to_string = "bsc-testnet", serialize = "binance-smart-chain-testnet")]
+    #[strum(to_string = "bsc", serialize = "bnb-smart-chain")]
+    #[cfg_attr(feature = "serde", serde(alias = "bsc", alias = "bnb-smart-chain"))]
+    BNBSmartChain = 56,
+    #[strum(to_string = "bsc-testnet", serialize = "bnb-smart-chain-testnet")]
     #[cfg_attr(
         feature = "serde",
-        serde(alias = "bsc_testnet", alias = "bsc-testnet", alias = "binance-smart-chain-testnet")
+        serde(alias = "bsc_testnet", alias = "bsc-testnet", alias = "bnb-smart-chain-testnet")
     )]
```

Comment:
I'd like to keep the enum variant name as as just because this is a breaking change, but add the additional bnb- names as additional aliases

Reply:
get it, I will revise this pr to `keep the original enums and add new enums`. thanks,

Reply:
updated